jumeg/connectivity/con_viz.py

```diff
@@ -657,54 +656,58 @@ def _get_circular_plot_labels(labels_mode, orig_labels, replacer_dict):
     return my_labels
 
 
-def _get_group_node_angles_and_colors(labels, orig_labels, node_order_size, cortex_colors=None):
-
-    if cortex_colors is None:
-        cortex_colors = ['m', 'b', 'y', 'c', 'r', 'g',
-                         'g', 'r', 'c', 'y', 'b', 'm']
-
+def _get_node_grouping(label_groups, orig_labels):
     ######################################################################
-    # Get labels in left and right hemisphere
+    # Get labels in left and right hemisphere in the right order for the circular plot
```

Comment:
Try use triple quotes to indicate docstrings for uniformity.

Reply:
In that case it was not intended as a docstring. A separate docstring would have to be added.